pages/telekonseling.tsx

```diff
@@ -0,0 +1,62 @@
+import { GetStaticProps } from "next";
+import { NextSeo } from "next-seo";
+import { BackButton } from "~/components/layout/back-button";
+import {
+  Page,
+  InternalPageContent,
+  InternalPageHeader,
+  // InternalPageSection,
+} from "~/components/layout/page";
+import { Container } from "~/components/ui/container";
+import { TelecounselingSection } from "~/components/telekonseling/telekonseling-link-section";
+import telekonseling, {
+  Telecounseling,
+} from "~/lib/content/telekonseling";
+
+interface TelecounselingPageProps {
+  telekonseling: Telecounseling;
+}
+
+const meta = {
+  title: `Curhat atau Telekonseling`,
+  description: `Kumpulan informasi mengenai donasi, nomor rekening, dan lainnya yang relevan dengan bantuan COVID-19.`,
```

Comment:
The description should represent the page.

Reply:
Ah, yes. I copied it from figma. Just update it on my last commit.